pkg/autoscaling/calculation.go

```diff
@@ -405,6 +403,16 @@ func findBestGroupToScaleOut(rc *cluster.RaftCluster, strategy *Strategy, scaleO
 		Component:    component.String(),
 		Count:        0,
 		ResourceType: resources[0].ResourceType,
+		Labels: []*metapb.StoreLabel{
+			{
+				Key:   groupLabelKey,
+				Value: autoScalingGroupLabelKeyPrefix + component.String(),
```

Comment:
it is better to leave a TODO here because we should make sure the group name is not duplicated when there are already other groups but PD still decides to create a new group.

Reply:
Done.